corehq/apps/linked_domain/util.py

```diff
@@ -102,6 +102,8 @@ def pull_missing_multimedia_for_app(app, old_multimedia_ids=None, force=False):
         media_to_pull = _get_missing_multimedia(app, old_multimedia_ids)
     remote_details = app.domain_link.remote_details
     fetch_remote_media(app.domain, media_to_pull, remote_details)
+    if force:
+        app.save()
```

Comment:
I don't love this, but it gives an easy way to fix apps that already have multimedia broken due to local conflict. As a user, I think I would assume that to "force pull" multimedia might also rewrite (and save) any broken multimedia references, which is what this is doing.